evcouplings/couplings/pairs.py

```diff
@@ -674,28 +690,39 @@ def probability(self, x, plot=False):
         # Calculate the minimum score
         min_score = abs(np.min(self.x))
 
-        return x / min_score
+        raw_evcomplex = x / min_score
+
+        # if we have N_effL, use it to calculate the corrected score
+        if self.N_effL:
+            return raw_evcomplex / (1+math.sqrt(self.N_effL))
+
+        # else return the raw score
+        return raw_evcomplex
 
 
 def add_mixture_probability(ecs, model="skewnormal", score="cn",
-                            clamp_mu=False, plot=False):
+                            clamp_mu=False, plot=False, N_effL=None):
```

Comment:
I would recommend neff_over_l as parameter name (also see comment in function itself)